### PR TITLE
Document AQ envs, and some more for AQ_NO_MODIFIERS

### PR DESCRIPTION
**Pull request by ikalco**
self explanatory

### CODE REVIEW

**Review by vaxerski — APPROVED** [review]

File: pages/Configuring/Environment-variables.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ environment on traditional Linux distros.
 - `HYPRLAND_NO_SD_NOTIFY=1` - If systemd, disables the `sd_notify` calls.
 - `HYPRLAND_NO_SD_VARS=1` - Disables management of variables in systemd and dbus activation environments.
 
+## Aquamarine Environment Variables <!-- ref https://github.com/hyprwm/aquamarine/blob/main/docs/env.md -->
+
+- `AQ_TRACE=1` - Enables more verbose logging.
+- `AQ_DRM_DEVICES=` - Set an explicit list of DRM devices (GPUs) to use. It's a colon-separated list of paths, with the first being the primary.
+  E.g. `/dev/dri/card1:/dev/dri/card0`
+- `AQ_NO_ATOMIC=1` - Disables drm atomic modesetting
```
Comment (vaxerski) [inline pages/Configuring/Environment-variables.md:48]: actually, please omit this

Comment (ikalco) [reply pages/Configuring/Environment-variables.md]: done :)

Comment (vaxerski) [reply pages/Configuring/Environment-variables.md]: I meant only that one var (no_atomic)

Comment (ikalco) [reply pages/Configuring/Environment-variables.md]: github is trolling lol
image

**Review by vaxerski — APPROVED** [review]

**Pull request merged**